manage.py

```diff
@@ -145,7 +167,7 @@ def add_fake_data(number_users):
     Adds fake data to the database.
     """
     User.generate_fake(count=number_users)
-
+    Listing.generate_fake_listings(count=number_users)
```

Comment:
Where is `generate_fake_listings`? And we should separate the commands to generate users vs listings, or at least not use `number_users` for both